includes/Core/Tags/First_Party_Mode/First_Party_Mode.php

```diff
@@ -137,4 +133,27 @@ public function healthcheck() {
 	public function on_admin_init() {
 		$this->cron->maybe_schedule_cron();
 	}
+
+	/**
+	 * Checks if an endpoint is healthy. The endpoint must return a `200 OK` response with the body `ok`.
+	 *
+	 * @since 1.141.0
```

Comment:
It would be good to include an additional `@since` declaration here indicating that this method has been relocated.

```suggestion
	 * @since 1.141.0
	 * @since n.e.x.t Relocated from REST_First_Party_Mode_Controller.
```